### DISCUSSION TITLE
How to turn off What's new?

### DISCUSSION THREAD

**Discussion by quagmt** [LazyVim/LazyVim#4989] [Q&A]
I updated LazyVim to v13 recently and noticed that it shows `What's New` quite frequently (even when a plugin has available update). How to prevent this from showing?
![image](https://github.com/user-attachments/assets/1d9e6f26-1457-4bff-9ba3-14f3590e19de)

**Comment by folke** [comment]
That only shows if that file changed, so it's not normal it shows all the time for you.
It's saved in lazyvim.json.
Check the docs to disable if you want

**Comment by quagmt** [comment]
@folke I think there's is something wrong with it. It usually shows `Migrating to lazyvim.json to version 7` on the right and the `What's News` page at the same time even though my `lazyvim.json` is already up-to-date
```
{
  "extras": [
    "lazyvim.plugins.extras.coding.mini-surround",
    "lazyvim.plugins.extras.ui.mini-animate",
    "lazyvim.plugins.extras.util.mini-hipatterns",
    "lazyvim.plugins.extras.vscode"
  ],
  "news": {
    "NEWS.md": "7429"
  },
  "version": 7
}
```
![image](https://github.com/user-attachments/assets/943417ce-8f05-4600-8825-f85e6c3343fd)

Can you help show where to turn this off? I already checked the docs but couldn't find it, maybe I'm missing something

**Reply by folke** [reply]
that can only happen if you overwrite or undo changes in lazyvim.json

**Reply by folke** [reply]
As for disabling https://www.lazyvim.org/configuration#default-settings
But I'd suggest you fix whatever issue you're having instead.
Those migrations can also mess up things.

**Reply by dpetka2001** [reply]
Maybe also check permissions for `lazyvim.json` if you can write to it? I remember older issues where NixOs users had similar issue. Not sure if it's relevant to this issue.

**Reply by quagmt** [reply]
Thanks, I'll check my settings then. 
However, this is quite weird. I never touch the `lazyvim.json` file.

**Reply by quagmt** [reply]
@dpetka2001 permission seems fine 
![image](https://github.com/user-attachments/assets/fcb5714b-47a4-4645-96ed-fcc2f2ad89c6)

Here's my full config: https://github.com/quagmt/lazyvim

**Reply by folke** [reply]
Looking at your git log of lazyvim.json, it all looks normal.
I don't see how it's possible that way that the what'snew would be shown when the news was not changed...

**Reply by dpetka2001** [reply]
I just cloned your dotfiles and tried your config and can't reproduce. Screencast following

[Screencast_2024-12-11-12-46-43.webm](https://github.com/user-attachments/assets/e780a7f7-328d-43af-98c1-9b5ad5a90865)

**Reply by quagmt** [reply]
I found the issue. It's because of the [vscode-neovim](https://github.com/vscode-neovim/vscode-neovim) extension that changes the `lazyvim.json` when it's loaded. Not sure how/why it does it tho.
[Screencast from 2024-12-11 18-04-19.webm](https://github.com/user-attachments/assets/1b2d3fa4-9e98-4cc8-b059-c5a35b4d69be)
